tests/test_admin/test_routers.py

```diff
@@ -0,0 +1,98 @@
+from fastapi import status
+from fastapi.testclient import TestClient
+
+from src.apps.user.schemas import UserOutputSchema
+from tests.test_orders.conftest import db_orders
+from tests.test_products.conftest import db_categories, db_products
+from tests.test_users.conftest import (auth_headers, create_superuser, db_user,
+                                       superuser_auth_headers)
+
+
+def test_superuser_can_get_all_superusers(
+    sync_client: TestClient, superuser_auth_headers: dict[str, str]
+):
+    response = sync_client.get("admin/superusers", headers=superuser_auth_headers)
+
+    assert response.status_code == status.HTTP_200_OK
+    assert response.json()["total"] == 1
+
+
+def test_new_superuser_can_get_all_superusers(
+    sync_client: TestClient,
+    superuser_auth_headers: dict[str, str],
+    db_user: UserOutputSchema,
+    auth_headers: dict[str, str],
```

Comment:
when staff role added, there should be only one superuser and some users as staff, so keep it in mind